app/util/analytics.py

```diff
@@ -188,6 +189,30 @@ def get_jira_version(self):
         jira_server_version = jira_server_info.get('version', '')
         return jira_server_version
 
+    def get_jira_nodes_count(self):
+        html_pattern = '<td><strong>Nodestate:</strong></td><td>Active</td>'
+        client = JiraRestClient(host=self.config_yml.server_url, user=self.config_yml.admin_login,
+                                password=self.config_yml.admin_password)
+        if self.get_application_version() >= '8.1.0':
+            jira_nodes_count_response = client.get_nodes_count_rest()
+            return len(jira_nodes_count_response)
```

Comment:
As jira_nodes_count_response is used only once, it would be nice to have a one-liner instead.